### DISCUSSION TITLE
SCAM repositories abusing 'fork's and 'star's

### DISCUSSION THREAD

**Discussion by goznauk** [community/community#129722] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Show & Tell

### Body

These repositories contain no valid code and only have a suspicious download link:

https://github.com/Aman12562/ByBitAB
https://github.com/nyancat-bombernub/ByBit-trading-bot
https://github.com/rangones/ByBit-trading-bot

Despite this, they have a lot of stars and forks, making them appear at the top of search results. All of these repositories share the same description, which leads me to believe this is a scam project abusing the GitHub system. We should be cautious and report these repositories.

**Comment by ebndev** [comment, marked as answer]
Hi @goznauk, 👋🏻 We really appreciate you flagging this. 

For anyone who may see this and would like to report repositories like this in the future, the best route to get this to the proper GitHub team is to use our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam). Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!
